Libraries/PeriphDrivers/Include/MAX32520/spi.h

```diff
@@ -643,6 +643,18 @@ void MXC_SPI_AbortAsync(mxc_spi_regs_t *spi);
  * @param   spi         Pointer to SPI registers (selects the SPI block used.)
  */
 void MXC_SPI_AsyncHandler(mxc_spi_regs_t *spi);
+
+/**
+ * @brief   Enable/Disable HW CS control feature.
+ *
+ * Depending on the application, the user might need to manually drive the slave select pin.
+ * The SPI driver automatically drives the SS pin and this function enables/disables this
+ * feature.
+ *
+ * @param   spi             Pointer to SPI registers (selects the SPI block used.)
+ * @param   state           true or false
```

Comment:
The description for the state variable should provide more detail. Please update this in all header files to something like, "Non-zero values: enable HW SS mode. Zero: disable HW SS mode."

Reply:
Added.